<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FileListWindow</class>
 <widget class="QMainWindow" name="FileListWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Hasher</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QFormLayout" name="formLayout_5">
    <property name="leftMargin">
     <number>12</number>
    </property>
    <property name="topMargin">
     <number>3</number>
    </property>
    <property name="rightMargin">
     <number>12</number>
    </property>
    <property name="bottomMargin">
     <number>12</number>
    </property>
    <item row="0" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_2"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuNist">
    <property name="title">
     <string>NIST</string>
    </property>
    <addaction name="actionItem_1"/>
    <addaction name="actionUpdate_NIST_data"/>
    <addaction name="actionRemove_NIST_data"/>
   </widget>
   <widget class="QMenu" name="menuScan">
    <property name="title">
     <string>Scan</string>
    </property>
    <addaction name="actionCreate_new_scan"/>
    <addaction name="actionRemove_scan"/>
   </widget>
   <addaction name="menuScan"/>
   <addaction name="menuNist"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QDockWidget" name="dbControlDock">
   <property name="features">
    <set>QDockWidget::DockWidgetFloatable|QDockWidget::DockWidgetMovable</set>
   </property>
   <property name="allowedAreas">
    <set>Qt::LeftDockWidgetArea|Qt::RightDockWidgetArea</set>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dbControlDockContents">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="sizeConstraint">
        <enum>QLayout::SetDefaultConstraint</enum>
       </property>
       <item>
        <widget class="QLineEdit" name="lineEdit">
         <property name="enabled">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="openButton">
         <property name="toolTip">
          <string>Open a new system database</string>
         </property>
         <property name="text">
          <string>Open...</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="createButton">
         <property name="text">
          <string>Create...</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QTabWidget" name="databaseTabs">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="tabShape">
        <enum>QTabWidget::Rounded</enum>
       </property>
       <property name="currentIndex">
        <number>0</number>
       </property>
       <property name="usesScrollButtons">
        <bool>false</bool>
       </property>
       <property name="tabsClosable">
        <bool>false</bool>
       </property>
       <property name="movable">
        <bool>false</bool>
       </property>
       <widget class="QWidget" name="scansTab">
        <attribute name="title">
         <string>Scans</string>
        </attribute>
        <layout class="QFormLayout" name="formLayout">
         <property name="fieldGrowthPolicy">
          <enum>QFormLayout::FieldsStayAtSizeHint</enum>
         </property>
         <item row="0" column="0">
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Scan:</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QComboBox" name="scansAvailableCombo">
           <property name="editable">
            <bool>false</bool>
           </property>
           <item>
            <property name="text">
             <string>No database loaded</string>
            </property>
           </item>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QPushButton" name="scanNowButton">
           <property name="text">
            <string>New scan...</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>Scan date:</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QLineEdit" name="scanDateLabel">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>Files:</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QLineEdit" name="scanFilesCountLabel">
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="4" column="1">
          <widget class="QPushButton" name="pushButton_4">
           <property name="text">
            <string>Remove scan...</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="settingsTab">
        <attribute name="title">
         <string>Settings</string>
        </attribute>
        <layout class="QFormLayout" name="formLayout_6">
         <property name="fieldGrowthPolicy">
          <enum>QFormLayout::FieldsStayAtSizeHint</enum>
         </property>
         <item row="0" column="0">
          <widget class="QLabel" name="NSRL">
           <property name="text">
            <string>NSRL:</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLabel" name="nsrlStatusLabel">
           <property name="text">
            <string>NSRL status</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QPushButton" name="nsrlAddUpdateButton">
           <property name="text">
            <string>Add NSRL data</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Base Path:</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QLineEdit" name="lineEdit_2">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
     <item>
      <widget class="QProgressBar" name="progressBar">
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="value">
        <number>24</number>
       </property>
       <property name="invertedAppearance">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
    <zorder>label_2</zorder>
    <zorder>lineEdit</zorder>
    <zorder>horizontalLayoutWidget_3</zorder>
    <zorder>databaseTabs</zorder>
    <zorder>progressBar</zorder>
   </widget>
  </widget>
  <action name="actionItem_1">
   <property name="text">
    <string>Add NIST data...</string>
   </property>
  </action>
  <action name="actionUpdate_NIST_data">
   <property name="text">
    <string>Update NIST data...</string>
   </property>
  </action>
  <action name="actionRemove_NIST_data">
   <property name="text">
    <string>Remove NIST data...</string>
   </property>
  </action>
  <action name="actionCreate_new_scan">
   <property name="text">
    <string>Create new scan...</string>
   </property>
  </action>
  <action name="actionRemove_scan">
   <property name="text">
    <string>Remove scan...</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
